.yamato/multiprocess-project-tests.yml

```diff
@@ -8,15 +8,15 @@
 singlenode_multiprocess_test_testproject_{{ editor }}_{{ platform.name }}:
   name : Multiprocess Tests - {{ editor }} on {{ platform.name }}
   agent:
-    type: {{ platform.type }}
+    type: {{ platform.type }}{% if platform.name == "ubuntu" %}::GPU{% endif %}
     image: {{ platform.image }}
     flavor: {{ platform.flavor}}
   commands:
     - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
 {% if editor != "trunk" %}
     - unity-downloader-cli -u {{ editor }} -c editor -w --fast
     - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr{% if platform.name == "win" %}.bat --output utr.bat{% endif %}{% if platform.name != "win" %} --output utr && chmod +x ./utr{% endif %}
-    - {{ platform.editorpath }} -projectpath testproject -batchmode -nographics -quit -logfile BuildMultiprocessTestPlayer.log -executeMethod Unity.Netcode.MultiprocessRuntimeTests.BuildMultiprocessTestPlayer.BuildRelease
+    - {{ platform.editorpath }} -projectpath testproject -batchmode -quit -logfile BuildMultiprocessTestPlayer.log -executeMethod Unity.Netcode.MultiprocessRuntimeTests.BuildMultiprocessTestPlayer.BuildRelease
```

Comment:
Is there a reason for removing -nographics, because we are running batchmode anyway.

Reply:
@ashwinimurt  On linux, in 2022.2 the Editor was crashing with a high frequency with errors that looked like something was still trying to init the graphics stack. With this change, that problem was completely removed. I can try a different image and see if the problem is any better since I didn't see this problem when I was using a custom image and not one of the standard CI images for Ubuntu.